src/scripts/auxLists/index.ts

```diff
@@ -1,12 +1,16 @@
+import { mapSupportedNetworks, SupportedChainId } from '@cowprotocol/cow-sdk'
 import { fetchAndProcessCoingeckoTokens } from './coingecko'
 import { fetchAndProcessUniswapTokens } from './uniswap'
-import { getCoingeckoTokenIdsMap } from './utils'
+import { getCoingeckoTokenIdsMap, OverridesPerChain } from './utils'
+
+const OVERRIDES: OverridesPerChain = mapSupportedNetworks({})
```

Comment:
This is what I've faced recently.
You should do `const OVERRIDES: OverridesPerChain = mapSupportedNetworks(() => ({}))`

<img width="648" alt="image" src="https://github.com/user-attachments/assets/d515178a-fdfe-4407-8f1b-814d0a2b8bb8" />

Reply:
```
(function() {
const ALL_SUPPORTED_CHAIN_IDS = [1, 100, 42161, 8453, 11155111]

function mapSupportedNetworks(value) {
  return ALL_SUPPORTED_CHAIN_IDS.reduce(
    (acc, chainId) => ({
      ...acc,
      [chainId]: typeof value === 'function' ? value(chainId) : value,
    }),
    {}
  )
}

const OVERRIDES = mapSupportedNetworks({})
OVERRIDES[8453]['0x18dd5b087bca9920562aff7a0199b96b9230438b'] = { decimals: 8 }

console.log('OVERRIDES', OVERRIDES)
})()
```